### PR TITLE
Adding additional highlighting syntax

### PR DESCRIPTION
**Pull request by n8felton**

### CODE REVIEW

File: syntaxes/flexlm-lic.tmLanguage.json [hunk 1]
```diff
@@ -34,13 +34,13 @@
...
 				"name": "variable.parameter.flexlm-lic",
-				"match": "\\b(BORROW|DUP_GROUP|FLOAT_OK|HOST_BASED|ISSUED|ISSUER|LINGER|NOTICE|OVERDRAFT|PLATFORMS|SN|START|SUITE_DUP_GROUP|SUPERSEDE|TS_OK|USER_BASED|VENDOR_STRING|HOSTID|SIGN)\\b"
+				"match": "\\b(BORROW|DUP_GROUP|FLOAT_OK|HOST_BASED|ISSUED|ISSUER|LINGER|NOTICE|OVERDRAFT|PLATFORMS|SN|START|SUITE_DUP_GROUP|SUPERSEDE|TS_OK|USER_BASED|VENDOR_STRING|HOSTID|SIGN|COMPONENTS)\\b"
 			}]
 		},
 		"constants": {
 			"patterns": [{
 				"name": "constant.language.flexlm-lic",
-				"match": "\\b(permanent)\\b"
+				"match": "\\b(permanent|\\d{2}-\\w{3}-\\d{4})\\b"
```
Comment (sekineh) [inline syntaxes/flexlm-lic.tmLanguage.json]: We often see the line like this where the day part is a single digit:
```
INCREMENT FEATURE_NAME daemonname 2020.07 8-jul-2020 1 \
    (line continues...)
```

Can you support the above pattern?

**Review by sekineh — CHANGES_REQUESTED** [review]

File: syntaxes/flexlm-lic.tmLanguage.json (hunk 1 above)
Comment (n8felton) [reply syntaxes/flexlm-lic.tmLanguage.json]: Updated to allow for single digit date

**Comment by sekineh** [comment]
Looks good! I'm going to merge this.
Thanks!

**Comment by sekineh** [comment]
I'll publish the change this week.

**Pull request merged**